package.json

```diff
@@ -54,6 +55,7 @@
     "eslint": "^2.9.0",
     "gulp-mocha": "^2.2.0",
     "sinon": "^1.17.4",
+    "tmp": "0.0.28",
```

Comment:
what's the difference between this an `temp`? @garlicnation is adding `temp` in #46